.drone.star

```diff
@@ -1,5 +1,16 @@
 def main(ctx):
     versions = [
+        {
+            "value": "10.9.0-beta1",
+            "qa": "https://download.owncloud.org/community/testing/owncloud-complete-20211118-qa.tar.bz2",
+            "tarball": "https://download.owncloud.org/community/testing/owncloud-complete-20211118.tar.bz2",
+            "tarball_sha": "4b7ab788358d1dbc96db44a6c232236e495ca5590701e6056c5ef2fac9e89a49",
+            "ldap": "https://github.com/owncloud/user_ldap/releases/download/v0.15.4/user_ldap-0.15.4.tar.gz",
+            "ldap_sha": "84412529b941723ef9523fe3c6d7c43cf2e44ab2522a9a9933966228f96122de",
+            "php": "7.4",
+            "base": "v20.04",
+            "tags": [],
```

Comment:
It should have at least one tag!? `10.9.0-rc1` should be the best fit when I l look at the image history https://hub.docker.com/r/owncloud/server/tags?page=1&name=rc

Reply:
That depends on what it is in core. If we have release it as rc use rc in the docker tag. If ee have released it as beta use beta see https://hub.docker.com/r/owncloud/server/tags?page=1&name=beta. However I agree on the missing tag 🙂

Reply:
right... it should be a beta

Reply:
Ive checked the starlak code. Tags are not required in this case as versions.value is used for tagging as well. As core was tagged as beta1 as well this PR should be fine.

Reply:
ok, good as well :+1: